devtools/enforcer-rules/src/it/smoketest/ext1/deployment/pom.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>io.quarkus</groupId>
+        <artifactId>quarkus-enforcer-rules-smoketest-ext1-parent</artifactId>
+        <version>1.0-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>quarkus-enforcer-rules-smoketest-ext1-deployment</artifactId>
```

Comment:
To be 100% accurate, the deployment module should depend on the runtime one.

Reply:
Good point, will add it!